spec/std/yaml/schema/core_spec.cr

```diff
@@ -114,10 +114,10 @@ describe YAML::Schema::Core do
   it_parses_scalar "2002-1-2T10:11:12", Time.new(2002, 1, 2, 10, 11, 12, kind: Time::Kind::Utc)
   it_parses_scalar "2002-1-2   10:11:12", Time.new(2002, 1, 2, 10, 11, 12, kind: Time::Kind::Utc)
   it_parses_scalar "2002-1-2   1:11:12", Time.new(2002, 1, 2, 1, 11, 12, kind: Time::Kind::Utc)
-  it_parses_scalar "2002-1-2T10:11:12.3", Time.new(2002, 1, 2, 10, 11, 12, 300, kind: Time::Kind::Utc)
-  it_parses_scalar "2002-1-2T10:11:12.34", Time.new(2002, 1, 2, 10, 11, 12, 340, kind: Time::Kind::Utc)
-  it_parses_scalar "2002-1-2T10:11:12.345", Time.new(2002, 1, 2, 10, 11, 12, 345, kind: Time::Kind::Utc)
-  it_parses_scalar "2002-1-2T10:11:12.3456", Time.new(2002, 1, 2, 10, 11, 12, 345, kind: Time::Kind::Utc)
+  it_parses_scalar "2002-1-2T10:11:12.3", Time.new(2002, 1, 2, 10, 11, 12, 300 * Time::NANOSECONDS_PER_MILLISECOND, kind: Time::Kind::Utc)
```

Comment:
Please **do not** use these `:nodoc:` constants from `Time`, they are nodoc for a reason.

Reply:
NANOSECONDS_PER_MILLISECOND  dropped from specs.